src/test/java/fi/otavanopisto/kuntaapi/test/AbstractIntegrationTest.java

```diff
@@ -712,6 +788,32 @@ protected String getServiceLocationChannelId(int index) throws InterruptedExcept
     return getServiceLocationChannelId(index, 3); 
   }
   
+  protected ServiceLocationServiceChannel getServiceLocationServiceChannel(int index, int waitCount) throws JsonParseException, JsonMappingException, IOException, InterruptedException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)